### PR TITLE
update lesson-01 material as agreed on review meeting

### PR DESCRIPTION
**Pull request by Aprillion**

### CODE REVIEW

File: lesson-0/intro_to_js.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Intro to JavaScript world
 
+Please ask questions if some topic is not clear and/or if I speak to slow or quiet.
+Purpose of this introduction is to give you vocabulary to ask better questions,
```
Comment (lspdv) [inline lesson-0/intro_to_js.md:4]: Comma on the end?

**Comment by lspdv** [comment]
Good work.

**Comment by jaroslav-kubicek** [comment]
Huge kudos to all, but shall I mention English will be appreciated? :-)

File: lesson-0/intro_to_js.md (hunk 1 above)
Comment (Aprillion) [reply lesson-0/intro_to_js.md:4]: that's what I get for multitasking, fixed in next PR

**Comment by lspdv** [comment]
Ok sorry. I edited and rewritten in english.

**Pull request merged**